src/Stepper/Step.tsx

```diff
@@ -7,6 +7,25 @@ import CheckMark from "./CheckMark";
 import { FlexBox } from "../flex";
 import Exclamation from "./Exclamation";
 
+enum StepState {
```

Comment:
Given our earlier discussion on enums vs. string-literal unions, any reason you decided to go with an enum here?

Reply:
No good reason. I copied this one from sd2 and didn't feel like changing it. I think it's worth changing though so thanks for bringing it up.

Reply:
the enum has been eliminated  🔪